addons/medical_treatment/XEH_postInit.sqf

```diff
@@ -21,3 +21,92 @@ if (isServer) then {
 [QGVAR(actionCheckPulseLocal), FUNC(actionCheckPulseLocal)] call CBA_fnc_addEventHandler;
 [QGVAR(actionCheckBloodPressureLocal), FUNC(actionCheckBloodPressureLocal)] call CBA_fnc_addEventHandler;
 [QGVAR(actionPlaceInBodyBag), FUNC(actionPlaceInBodyBag)] call CBA_fnc_addEventHandler;
+
+// Notification lets the player known if they're being treated
+[QGVAR(initiated), {
+    params ["_caller", "_target"];
+
+    // Notifications don't make sense for self-healing or when dead
+    if (_target == _caller || {!alive _target}) exitWith {};
+
+    // Tracking who is responding lets us show notifications in future too
+    private _responders = _target getVariable [QGVAR(responders), []];
+    _responders pushBackUnique _caller;
+    _target setVariable [QGVAR(responders), _responders];
+
+    // Player only cares if they are the patient
+    if (_target == ACE_player) then {
```

Comment:
One further suggestion;

let's move the parts from l38 through l46, and the related logic in the other event handlers into the feedback component by raising a local event in here, and listing for that event in the feedback component.

I realise that we are shuffling around a bit of logic here,  but this will make the `responders` functionality introduced generic and makes the notifications bit part of a component that is more suitable for such functionality.